plumbing/clock/clock.go

```diff
@@ -40,3 +41,7 @@ func NewConfiguredBlockClock(blockTime time.Duration) *DefaultBlockClock {
 func (bc *DefaultBlockClock) BlockTime() time.Duration {
 	return bc.blockTime
 }
+
+func (bc DefaultBlockClock) EpochSeconds() int64 {
```

Comment:
nit: this name seems a little unclear (could just be me).  Why not `Now`?

Reply:
Missing function comment

Reply:
Called it `EpochSeconds` due to this comment: https://github.com/filecoin-project/go-filecoin/pull/2882#discussion_r290123641

Reply:
will add a comment as well

Reply:
oh I see.  One reason I don't like `EpochSeconds` is that `epoch` is a term with a precise meaning for the consensus protocol and it appears to be used in a different way here.  This might be something @anorth didn't think of.